devbox.json

```diff
@@ -41,8 +41,5 @@
       "export PATH=\"$HOME/.cargo/bin:$PATH\"",
       "type unset 2>/dev/null && unset GOROOT"
     ]
-  },
-  "nixpkgs": {
```

Comment:
This field was deprecated by devbox sometime last year - as far as I can tell, it wasn't actually doing anything 😃